README.md

```diff
@@ -96,24 +96,48 @@ sudo dnf copr enable atim/leftwm -y && sudo dnf install leftwm
 
 ## Using a graphical login such as LightDM, GDM, LXDM, and others
 
-1) Copy leftwm.desktop to /usr/share/xsessions
-2) Create a symlink to the build of leftwm so that it is in your path:
+1. Dependencies: Rust, Cargo
+2. Clone the repository and change directory into it.
 
 ```bash
-cd /usr/bin
-sudo ln -s PATH_TO_LEFTWM/target/debug/leftwm
-sudo ln -s PATH_TO_LEFTWM/target/debug/leftwm-worker
-sudo ln -s PATH_TO_LEFTWM/target/debug/leftwm-state
-sudo ln -s PATH_TO_LEFTWM/target/debug/leftwm-check
+git clone https://github.com/leftwm/leftwm.git
+cd leftwm
 ```
 
-and
+3. Build leftwm
 
 ```bash
-sudo cp PATH_TO_LEFTWM/leftwm.desktop /usr/share/xsessions
+cargo build --release
+```
+
+4. Make a new share directory for leftwm
+
+```bash
+sudo mkdir -p /usr/share/leftwm
 ```
 
-You should now see LeftWM in your list of available window managers.
+5. Copy build files to the new share directory
+
+```bash
+sudo cp -r ./target/release/* /usr/share/leftwm
+```
+
+6. Create the symlinks
+
+```bash
+sudo ln -s /usr/share/leftwm/leftwm /usr/bin/leftwm
+sudo ln -s /usr/share/leftwm/leftwm-worker /usr/bin/leftwm-worker
+sudo ln -s /usr/share/leftwm/leftwm-state /usr/bin/leftwm-state
+sudo ln -s /usr/share/leftwm/leftwm-check /usr/bin/leftwm-check
+```
```

Comment:
To be honest, I don't know much about rust and thought those *.d  file were required for the executables to run correctly.  (My mistake)  Given that, copying files to /usr/share does seem unnecessary.  

Copying the executables to /usr/bin  makes more sense to me than symlinking.  Some users will build somewhere in their home directory and if they ever reorganize their home directory structure, it will break the symlinks and break leftwm.   Is this sound reasoning?  Or would you prefer to symlink?

Reply:
I agree on copying the files seems better.

Reply:
I agree, that copying and symlinking is kind of redundant. I think symlinking to the build directory is the better way, since it allows effort less rebuolds/updates.

Reply:
haha, awkward overlapping comments :)

Maybe have a sidenote with the symlink instructions, on the downsides? Or promote coping and a sidenote to mention the symlink possibility..

Reply:
I kinda on both sides, as symlinking means when you update the files should update to the new ones. However, at least for me, my build directories are a constant and my be overwritten or moved, etc. which makes it possible the symlinks break.

Reply:
another thought on the copy-method: this resembles kind of more like a proper install, whereas the symlink method is more like a development solution.
So maybe have two sections *Manual isntall* and *Dev Install*

Reply:
Ok.  I'll make separate sections for two different types of manual installs

Reply:
I've verified the installation instructions by testing on my laptop.  

I just have one point I want to verify:  Is a soft reload enough to load an updated version of the leftwm executable?  Or do you need to completely log out/in for the new executable to be read from disk?   I'm not sure if a soft reload retains an old executable in memory or if it gets swapped out dynamically.

Reply:
Im not 100% on this, however I think you have to logout to reload the binaries.

Reply:
The only difference between a hard and a soft reload is that the soft-reload tries to store (and reload afterwards) the state of leftwm in a temporary file and then actually go into a hard-reload.
In a hard-reload the leftwm-worker kills itself (the leftwm is basically just a launcher for the worker) and gets launched again (I believe by the loop in leftwm parent process).
At least with debug builds it definitely launches with the new binaries and I don't have any reason to believe the --release build behaves any different.

Reply:
Yep, @VuiMuich has it right, most of the work with leftwm is done by leftwm-worker. Leftwm will re-start leftwm-worker when it dies, so `pkill leftwm-worker` or a soft/hard reload effectively restarts leftwm-worker and therefore the “meat” of leftwm. 

Admittedly I’m not certain whether that also restarts leftwm-state if it’s already running, so might want to look into that. However, a restart does also up/down most of the programs that would be accessing leftwm-state so I imagine that ought to take care of that.

Reply:
> Admittedly I’m not certain whether that also restarts leftwm-state if it’s already running, so might want to look into that. However, a restart does also up/down most of the programs that would be accessing leftwm-state so I imagine that ought to take care of that.

Isn't `leftwm-state` just a standalone executable to process the conecnt of the socket file ind XDG_RUNTIME_DIR to provide this data on demand e.g. like when starting a status bar (or when I abuse it for 'debugging' issue reports :wink:).

Reply:
Thinking about this now, should we use install instead of copy like is recommended for aur packages, e.g. what is currently used in the aur package, `install -Dm755 leftwm leftwm-worker leftwm-state -t "$pkgdir"/usr/bin`.

Reply:
>Isn't leftwm-state just a standalone executable to process the content of the socket file ind XDG_RUNTIME_DIR

Good point. Yeah, it is. I guess if you're updating leftwm-state you might want to make sure it gets restarted but otherwise it's just a wrapper enit? 
>should we use install instead of copy like is recommended for aur packages

Install ensures the permissions are correct so that is an intriguing proposition from that perspective. I think this is a bit of a dev/release split though, as I would prefer links on the partition where I test stuff, but probably would prefer install on my stable partition. I suppose many would like that distinction as well. 

We also might look into the `-s` label to strip binaries down - might make LeftWM a little faster, idk.

Reply:
Here is the file size difference between stripped and unstripped binaries

| Name          | Size Unstripped | Size Stripped |
|---------------|-----------------|---------------|
| leftwm        | 3.4M            | 396K          |
| leftwm-check  | 4.7M            | 1.5M          |
| leftwm-state  | 7.6M            | 3.7M          |
| leftwm-worker | 7.3M            | 3.4M          |

Reply:
Awesome, thanks for this whole PR! Everything looks good to me. It's amazing how small LeftWM can get!